### PR TITLE
Add some travel destinations for fall

### PR DESCRIPTION
**Pull request by angiekollmer**
Please review this, @casey-collab

### CODE REVIEW

File: english/travel-destinations/fall/fall.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Fall is the time to enjoy nature's amazing color displays known as "fall foliage"!
+
+Two great national parks for leaf peeping are 
+the [Blue Ridge Parkway](http://www.nps.gov/blri/index.htm) 
+and [Acadia](http://www.nps.gov/acad/index.htm)
```
Comment (casey-collab) [inline english/travel-destinations/fall/fall.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by angiekollmer** [comment]
Please review this again, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :wind_chime:

**Pull request merged**